README.md

```diff
@@ -46,7 +46,8 @@ To do so:
           instead of Docker.
     
           **Note** that it requires manual configuration, such as installing our
-          shared plugins and any theme code to be tested.
+          shared plugins and any theme code to be tested. See [this Confluence article
+          for instructions](https://confluence.pmcdev.io/display/ENG/Getting+Started+With+PHPUnit+Tests#GettingStartedWithPHPUnitTests-RunningYourTestLocally).
```

Comment:
This should use the short URL that Confluence provides, otherwise any change to the document title will break the link.

Reply:
I made the Confluence URL change and added another troubleshooting item